src/nl/hannahsten/texifyidea/util/Labels.kt

```diff
@@ -51,15 +54,17 @@ fun PsiFile.findLatexLabelStringsInFileSetAsSequence(): Sequence<String> {
  */
 fun PsiFile.findLatexLabelPsiElementsInFileAsSequence(): Sequence<PsiElement> = sequenceOf(
     findLabelingCommandsInFileAsSequence(),
-    LatexParameterLabeledEnvironmentsIndex.getItems(this).asSequence()
+    LatexParameterLabeledEnvironmentsIndex.getItems(this).asSequence(),
+    LatexParameterLabeledCommandsIndex.getItemsInFileSet(this).asSequence()
```

Comment:
I think this should be `getItems` instead of `getItemsInFileSet`?

Reply:
You are totally right.